components/button/index.css

```diff
@@ -91,68 +89,39 @@ governing permissions and limitations under the License.
 
 }
 
-
 .spectrum-Button {
   @inherit: %spectrum-BaseButton;
   @inherit: %spectrum-ButtonWithFocusRing;
 
-
   border-radius: var(--mod-button-border-radius, var(--spectrum-button-border-radius));
   border-width: var(--mod-button-border-width, var(--spectrum-button-border-width));
   border-style: solid;
-  block-size: auto;
+  font-size: var(--mod-button-font-size, var(--spectrum-button-font-size));
+  font-weight: var(--mod-bold-font-weight, var(--spectrum-bold-font-weight));
+  gap: var(--spectrum-button-padding-label-to-icon);
   height: var(--mod-button-height, var(--spectrum-button-height));
-
-  min-width: var(--mod-button-min-width, var(--spectrum-button-min-width));
-  min-block-size: var(--mod-button-min-width, var(--spectrum-button-min-width));
-  position: relative;
-
-
-  padding-block-start: 0;
-  padding-block-end: 0;
-
-  position: relative;
-
   min-inline-size: var(--mod-button-min-width, var(--spectrum-button-min-width));
   min-block-size: var(--mod-spectrum-button-height, var(--spectrum-button-height));
-
-  height: var(--mod-button-height, var(--spectrum-button-height));
-  border-radius: var(--mod-button-border-radius, var(--spectrum-button-border-radius));
-  border-width: var(--mod-button-border-width, var(--spectrum-button-border-width));
   /* Start with text-only padding */
-  padding-inline-start: var(--mod-button-edge-to-visual, var(--spectrum-button-edge-to-visual));
-  padding-inline-end: var(--mod-button-edge-to-text, var(--spectrum-button-edge-to-text));
-
-
-  font-size: var(--mod-button-font-size, var(--spectrum-button-font-size));
-  font-weight: var(--mod-bold-font-weight, var(--spectrum-bold-font-weight));
+  padding-inline: var(--mod-button-edge-to-text, var(--spectrum-button-edge-to-text));
+  position: relative;
+  
+  /* let staticColor variants inherit their color */
+  color: inherit;
 
   &:hover,
   &:active {
     box-shadow: none;
   }
 
-  /* let staticColor variants inherit their color */
-  color: inherit;
-
   .spectrum-Icon {
-    inset-inline-end: calc(var(--mod-spectrum-button-edge-to-visual, var(--spectrum-button-edge-to-visual)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));
-    inset-block-end: calc(var(--mod-spectrum-button-edge-to-visual, var(--spectrum-button-edge-to-visual)) - var(--mod-button-border-width, var(--spectrum-button-border-width)));
+    margin-inline-start: calc(
```

Comment:
Ensures that the icon is closer to the edge of the Button (for now) that the label is by itself.